packages/components/src/List/Toolbar/Toolbar.scss

```diff
@@ -26,6 +26,9 @@ $tc-list-toolbar-background-color: $concrete !default;
 	:global(.navbar-nav) {
 		li a {
 			color: $black;
+			&:global(.dropdown-toggle) {
+				text-transform: capitalize;
+			}
```

Comment:
Why did you need to add a global `text-transform`?

Reply:
if i'm correct its because the top class is scoped locally, could have wrote it that way though : 

:global {
		.navbar-nav li a {
			color: $black;
			&.dropdown-toggle {
				text-transform: capitalize;
			}
		}
	}

which might be nicer, what do you think ?